includes/feeds/google.php

```diff
@@ -123,6 +123,36 @@ public function __construct( $calendar = '', $load_admin = true ) {
 		}
 	}
 
+	/**
+	 * Google API Client.
+	 *
+	 * @since  3.0.0
+	 * @access private
+	 *
+	 * @return \Google_Client
+	 */
+	private function get_client() {
+
+		$client = new \Google_Client();
+		$client->setApplicationName( 'Simple Calendar' );
+		$client->setScopes( $this->google_client_scopes );
+		$client->setDeveloperKey( $this->google_api_key );
+		$client->setAccessType( 'online' );
+		$client->setHttpClient(
+			new \GuzzleHttp\Client(
+				array(
+					'defaults' => array(
+						'headers' => array(
+							'Referer' => site_url(),
```

Comment:
Do you think we'd benefit at all by making this filterable? I know `site_url()` is filterable.. but maybe there is a use case where someone would want to only filter this specific use of it?